### DISCUSSION TITLE
Raising the map search slider

### DISCUSSION THREAD

**Discussion by Rndeez** [ppy/osu#26691] [Ideas] [answered]
I the slider to look through maps a lot and i find it really annoying to click it when you are on the final song at the bottom since the slider is hidden behind the osu! logo that grows bigger and hides it fully when it's being hovered over with the cursor. So i think that it'd 
good if it was raised a bit so that it'd be easier to use.

![image](https://github.com/ppy/osu/assets/112340391/c0ee906d-77e5-4c8a-9cdb-d0122bde209f) 
![image](https://github.com/ppy/osu/assets/112340391/da44641d-9ed0-451f-82f6-c857cedf3361)

**Comment by peppy** [comment, marked as answer]
Should be addressed by https://github.com/ppy/osu/pull/26701
